src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { SelectOption } from '../SelectOption/SelectOption';
+import { TextField } from '../TextField/TextField';
+
+interface Props {
+  onChangeTodo: (title: string, userId: number, completed?: boolean) => void;
+}
```

Comment:
It will be more readable, right?
```suggestion
interface Props {
  onChangeTodo: (
    title: string, 
    userId: number, 
    completed?: boolean
  ) => void;
}
```